cmake/oss.cmake.in

```diff
@@ -309,7 +309,7 @@ ExternalProject_Add(kafka-codegen-pex
   BUILD_COMMAND ""
   INSTALL_COMMAND
   COMMAND ${CMAKE_COMMAND} -E chdir <SOURCE_DIR> python3 -m venv env
-  COMMAND <SOURCE_DIR>/env/bin/pip install pex
+  COMMAND <SOURCE_DIR>/env/bin/pip install pex=2.1.100
   COMMAND <SOURCE_DIR>/env/bin/pex jsonschema jinja2 -o <INSTALL_DIR>/bin/kafka-codegen-venv)
```

Comment:
I think we should pin jsonschema and jinja2 as well

Reply:
discussed with ivo and i'll create a followup PR to pin the others